SYCL/ESIMD/api/functional/type_coverage.hpp

```diff
@@ -267,7 +273,11 @@ template <tested_types required> auto get_tested_types() {
                                         "unsigned int", "unsigned long",
                                         "unsigned long long");
     }
+#else
```

Comment:
Nit: let's start from the shorter path in general. By using `#if !defined` for example. In such case there will be no many lines between `#if` and `#else`, so it would be easier to read the logic